src/SpaceTime/Squared.cs

```diff
@@ -78,6 +78,12 @@ public static Squared<T> FromValue(float value)
         public int CompareTo(Squared<T> other) => value.CompareTo(other.value);
 
         #endregion
+        
+        #region tostring
+
+        public override string ToString() => $"|{value}|";
```

Comment:
I couldn't find a way to get to the `ToString` method from the `T` type. Feel free to let me know if you have any ideas.